src/fizzBuzz.js

```diff
@@ -0,0 +1,20 @@
+export default function fizzBuzz() {
+  var numbers = []
+  for (var number = 1; number <= 100; number++) {
+
+    if (number % 3 == 0 && number % 5 == 0){
+      numbers.push('FizzBuzz')
+
+    } else if (number % 5 == 0 && number % 3 !== 0) {
+          numbers.push('Buzz')
+
+      } else if (number % 3 == 0) {
+            numbers.push('Fizz')
+
+          } else {
+            numbers.push(number)
```

Comment:
you do not need to indent every new else if statement. Keeping the entire if else block on the same "level" would make it clearer and more easy to read.